spec/unit/default_repo.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
```

Comment:
ok. Note that rspec fails after this, and I don't know yet how to fix it; I will look into it tomorrow.